### PR TITLE
[Documentation][Book] Invoices

### PR DESCRIPTION
**Pull request by CoderMaggie**
| Q               | A
| --------------- | -----
| Branch?         | 1.6
| Doc fix?        | no
| New docs?    | yes
| License         | MIT

In order to continue describing the official Sylius plugins in the docs, I am adding a section about the [Sylius/InvoicingPlugin](https://github.com/Sylius/InvoicingPlugin).

localhost_63342_Sylius_docs_build_book_orders_invoices html

### CODE REVIEW

File: docs/book/orders/invoices.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+or if it was already paid). From the shop's point of view, an invoice is a sales invoice. From the customer's point of view,
+an invoice is a purchase invoice.
+
+Invoicing in Sylius
+-------------------
+
+Sylius is providing invoicing engine via a `free open-source plugin. <https://github.com/Sylius/InvoicingPlugin>`_
+Th plugin installation guide you will find in the plugin's README.
+
+Having the plugin installed you will notice a new main menu item among "Sales" section - "Invoices". It allows you to acces the index
```
Comment (ternel) [inline docs/book/orders/invoices.rst]:
```suggestion
Having the plugin installed you will notice a new main menu item among "Sales" section - "Invoices". It allows you to access the index
```

File: docs/book/orders/invoices.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+An Invoice is a commercial document issued by the shop, that is a sort of confirmation of the sale transaction. It indicates the
+products, quantities, agreed prices for these products. An invoice contains usually also payment terms (like due date,
+or if it was already paid). From the shop's point of view, an invoice is a sales invoice. From the customer's point of view,
+an invoice is a purchase invoice.
+
+Invoicing in Sylius
+-------------------
+
+Sylius is providing invoicing engine via a `free open-source plugin. <https://github.com/Sylius/InvoicingPlugin>`_
+Th plugin installation guide you will find in the plugin's README.
```
Comment (ternel) [inline docs/book/orders/invoices.rst]:
```suggestion
The plugin installation guide you will find in the plugin's README.
```

File: docs/book/orders/invoices.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+or if it was already paid). From the shop's point of view, an invoice is a sales invoice. From the customer's point of view,
+an invoice is a purchase invoice.
+
+Invoicing in Sylius
+-------------------
+
+Sylius is providing invoicing engine via a `free open-source plugin. <https://github.com/Sylius/InvoicingPlugin>`_
+The plugin installation guide you will find in the plugin's README.
+
+Having the plugin installed, you will notice a new main menu item among "Sales" section - "Invoices". It allows you to acces the index
```
Comment (GSadee) [inline docs/book/orders/invoices.rst]:
```suggestion
Having the plugin installed, you will notice a new main menu item among "Sales" section - "Invoices". It allows you to access the index
```

**Comment by GSadee** [comment]
Thank you, Magdalena! :1st_place_medal:

**Pull request merged**